app/models/user.rb

```diff
@@ -1,21 +1,5 @@
 class User < ActiveRecord::Base
-  has_many :short_urls
-  has_many :original_urls, through: :short_urls
-  def increment_total_clicks
-    self.total_clicks += 1
-    save
-  end
-
-  def trending_shotlink
-    max_shotlink = { clicks: 0 }
-    short_urls.each do |s|
-      if s.original_url.clicks > max_shotlink[:clicks]
-        max_shotlink[:clicks] = s.original_url.clicks
-        max_shotlink[:original_url] = s.original_url
-      end
-    end
-    max_shotlink
-  end
+  has_many :links
```

Comment:
### Code Quality - NitPick

By merging `OriginalUlr` with `ShortUrl` to `Links`, only one association is necessary.